config/job_conf.xml.sample_advanced

```diff
@@ -283,7 +283,11 @@
             <param id="insecure">true</param>
             <!-- True to communicate with Chronos over HTTPS; false otherwise-->
         </plugin>
-
+        <!-- Additionally any plugin or destination (below) may define an "enabled" parameter that should
+             evaluate to True or False. When setup using
+                 <param id="enabled" from_environ="<VAR>">DEFAULT</param>
```

Comment:
I think that makes it more clear, otherwise I could assume DEFAULT is a special keyword.